lineapy/utils/analytics/usage_tracking.py

```diff
@@ -33,6 +34,17 @@ def _py_version():
     )
 
 
+@lru_cache(maxsize=1)
+def _runtime():
+    if options.get("runtime") is not None:
+        return str(options.get("runtime"))  # Return manually set value
```

Comment:
see below. i prefer to add the config piece of runtime as a prefix and still get the original output of get_ipython in our tracker.

Reply:
Im not sure if I follow what you mean as a prefix, but I think we can just remove these two lines? It has nothing to do with options.

Reply:
I think @lionsardesai is suggesting an optional custom tag (which @mingjerli suggested as a way to "flag" events to ignore, e.g., those from dev work) being prepended to the runtime info. For instance, for my executions for certain demo session, I can append all runtime info with `sangyoon_demo`, in which case the runtime value (in event metadata) may read like `[sangyoon_demo]colab` instead of `colab`.